EmbeddedScripts.CSharp/Scripting/ScriptCodeRunner.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Linq;
+using Microsoft.CodeAnalysis.CSharp.Scripting;
+using System.Threading.Tasks;
+using EmbeddedScripts.CSharp.Common;
+using EmbeddedScripts.Shared;
+using Microsoft.CodeAnalysis.Scripting;
+
+namespace EmbeddedScripts.CSharp.Scripting
+{
+    public class ScriptCodeRunner : ICodeRunner
```

Comment:
Ideally there should be 1 ICodeRunner implementation per project. Let's split this project into 2 separate: Csharp.Roslyn.Scripting and Csharp.Roslyn for example. (Small code duplication is fine now, if it will be growing then new Roslyn.Shared project can be introduced